data/DALTON/basicDALTON-2015/Trp_polar_abalnr.dal

```diff
@@ -0,0 +1,63 @@
+BASIS
+STO-3G
+tryptophan static polarizability
```

Comment:
It's fine. You've already put a lot of work into tryptophan so I would keep it, especially since the coverage is really good.